lnd_test.go

```diff
@@ -556,6 +702,8 @@ func testSingleHopInvoice(net *networkHarness, t *harnessTest) {
 		t.Fatalf("unable to add invoice: %v", err)
 	}
 
+	time.Sleep(time.Millisecond * 200)
```

Comment:
Why was this sleep, and the one below it added? Either they should be removed or just increase the sleep times on them. 

Ideally, we need to get ride of all instances of `time.Sleep` within the tests .